xml/System.IO/DriveInfo.xml

```diff
@@ -117,22 +117,20 @@
         <Parameter Name="driveName" Type="System.String" />
       </Parameters>
       <Docs>
-        <param name="driveName">A valid drive path or drive letter. This can be either uppercase or lowercase, 'a' to 'z'. A null value is not valid.</param>
+        <param name="driveName">A valid drive letter or fully qualified path.</param>
         <summary>Provides access to information on the specified drive.</summary>
         <remarks>
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- Use this class to obtain information on drives. The drive name must be either an uppercase or lowercase letter from 'a' to 'z'. You cannot use this method to obtain information on drive names that are `null` or use UNC (\\\server\share) paths.  
+ Use this class to obtain information on drives. You cannot use this method to obtain information on drive names that are `null` or use UNC (\\\server\share) paths.  
+
+ On Windows, passing a mounted directory (e.g. C:\NetworkMountedDrive) returns the root drive information (e.g. C:).
```

Comment:
The constructor doesn't technically return anything, so this should be reworded.